client/src/components/Tables/UserRecords/DeniedUsers/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { FC, ReactElement, ReactNode, useEffect, useState } from 'react';
+import { IUser } from 'ssw-common';
+import toast from 'react-hot-toast';
+
+import DynamicTable from '../../DynamicTable';
+import ReviewUserModal from '../../../Modals/ReviewUser';
+import { nameColumn, roleColumn, userColumn, teamsColumnModal } from '../utils';
+import { isError, updateOnboardingStatus } from '../../../../api';
+import { onboardingStatusEnum } from '../../../../utils/enums';
+import { buildColumn } from '../../DynamicTable/util';
+
+interface DeniedUserProps {
+  users: IUser[];
+}
+
+const DeniedUsers: FC<DeniedUserProps> = ({ users }): ReactElement => {
+  const [data, setData] = useState<IUser[]>(users);
+
+  const updateUserStatus = async (
```

Comment:
Seems like `updateUserStatus` is not used here. Maybe make `actionUpdate` of `ReviewUserModal` optional?